### ISSUE TITLE
Enhancement: Add Biostage to non-generic binary list

### ISSUE THREAD

**Issue by Kinkojiro** [ME3Tweaks/LegendaryExplorer#21]
Biostage layout:
Length (int)
Name: m_aCameraList + index 0 + int unknown
Count + int unknown
[Camera name + index
property name  + floatproperty + length +float value (repeated like:
fPitchDelta name + floatproperty + length +float value
fYawDelta name + floatproperty + length +float value)
None]

Example: Biostage #82 in BioD_Nor_300_Crit.pcc

**Comment by Mgamerz** [comment]
Implemented basic support in e4b6a46452cafbcb37a898f5ef53d37e2d6581bd

**Comment by Mgamerz** [comment]
Data format, for anyone interested in future:

```
Length (int) [4]
Cameraname (Name) [8]
Count (int64) [8]

The following data format repeats as an embedded binary struct. There are count number of structs.
Camera (Name) [8]

The following items will repeat until None is found.
Property (Name) [8]
FloatProperty (Name) [8] (literally "FloatProperty")
Length (int64) [8]
FloatValue (float) [4]

None (Name) [8]
```
